UI/window-basic-interaction.cpp

```diff
@@ -215,6 +215,42 @@ static int TranslateQtMouseEventModifiers(
 	return modifiers;
 }
 
+static int TranslateQtNavigationKeys(
+	QKeyEvent *event)
+{
+	int navkeys = NAVKEY_NONE;
+
+	switch (event->key())
+	{
```

Comment:
Style:  Place this curly brace at the end of the preceding line with a space before it.